From 73ded33c53d5c358a50ece801c57fcbac0cffad2 Mon Sep 17 00:00:00 2001
From: Alistair Strachan <astrachan@google.com>
Date: Fri, 7 Dec 2018 11:50:07 -0800
Subject: ANDROID: cuttlefish_defconfig: Enable
 CONFIG_ARM64_LSE_ATOMICS

Enabling this was previously blocked by a lack of support for this
feature in clang, but that problem has been resolved in a newer version
of the compiler.

Bug: 120439617
Change-Id: I0f5fd2439c5a71ee0988648970576b46b2c4d20b
Signed-off-by: Alistair Strachan <astrachan@google.com>
---
 arch/arm64/configs/cuttlefish_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/configs/cuttlefish_defconfig b/arch/arm64/configs/cuttlefish_defconfig
index 8d379eb47d343..3c799383ff401 100644
--- a/arch/arm64/configs/cuttlefish_defconfig
+++ b/arch/arm64/configs/cuttlefish_defconfig
@@ -45,7 +45,6 @@ CONFIG_SWP_EMULATION=y
 CONFIG_CP15_BARRIER_EMULATION=y
 CONFIG_SETEND_EMULATION=y
 CONFIG_ARM64_SW_TTBR0_PAN=y
-# CONFIG_ARM64_LSE_ATOMICS is not set
 CONFIG_RANDOMIZE_BASE=y
 CONFIG_CMDLINE="console=ttyAMA0"
 CONFIG_CMDLINE_EXTEND=y
-- 
2.23.0.444.g18eeb5a265-goog

